enhancements/disaster-recovery/automated-cluster-backups.md

```diff
@@ -0,0 +1,324 @@
+---
+title: automated-cluster-backups
+authors:
+  - "@marun"
+reviewers:
+  - "@hexfusion"
+  - "@lilic"
+  - "@deads2k"
+  - "@smarterclayton"
+approvers:
+  - "@hexfusion"
+  - "@lilic"
+  - "@deads2k"
+  - "@smarterclayton"
+creation-date: 2021-07-08
+last-updated: 2021-07-08
+status: implementable
+see-also:
+  - "https://docs.google.com/document/d/1LyZQMgLvgd81iQndcDcjAVN25ZEhCxVGMttoiRkSU6g/edit#"
+  - "https://docs.openshift.com/container-platform/4.7/backup_and_restore/backing-up-etcd.html"
+  - "https://rancher.com/docs/rancher/v2.x/en/backups/v2.5/configuration/backup-config/"
+  - "https://www.openshift.com/blog/ocp-disaster-recovery-part-1-how-to-create-automated-etcd-backup-in-openshift-4.x"
+---
+
+# Automated Cluster Backups
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Enable the configuration of automated cluster backups.
```

Comment:
needs specificity.  velero promises the same.  This is specifically about backing up etcd state and NOT about backing PVs or node state, correct?

Reply:
Thanks to 4.x being self-hosted, it's not just about etcd backups but also content from /etc/kubernetes, but to your point I'll clarify that.